bin/run_celery_worker_log_parser

```diff
@@ -16,8 +16,8 @@ fi
 
 exec newrelic-admin run-program celery -A treeherder worker \
     -Q \
-parse_job_logs,\
-parse_job_logs_fail,\
+parse_job_log,\
+parse_job_log_fail,\
```

Comment:
Good point about orphaned queues. Will fix.

Do we actually need to adjust Procfile? I don't see any reference to `parse_job_logs` there.